cli-plugins/hooks/printer.go

```diff
@@ -0,0 +1,18 @@
+package hooks
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/morikuni/aec"
+)
+
+func PrintNextSteps(out io.Writer, messages []string) {
+	if len(messages) == 0 {
+		return
+	}
+	fmt.Fprintln(out, aec.LightBlueF.Apply("\nNext steps:"))
+	for _, n := range messages {
+		_, _ = fmt.Fprintf(out, "    - %s\n", n)
+	}
```

Comment:
Current implementation of hooks looks like this:
<img width="669" alt="image" src="https://github.com/docker/cli/assets/1542321/466c14d4-c336-45ed-9831-ebe114be3ef0">

And with multiple messages:
<img width="1074" alt="image" src="https://github.com/docker/cli/assets/1542321/7e5c1936-084f-45d2-901b-bfa0b720ec4f">

Basically two main differences:
- `Next steps:` vs `What's Next?` and colors
- `-` in front of each lines

I can change the scout CLI to add `-` on each line, so we ensure it's done the same way everywhere.
About the title I wonder if `What's next?` isn't a bit better. But I'll defer to @jalonsogo